lib/entry-data-plugin.js

```diff
@@ -69,15 +73,11 @@ export default function entryDataPlugin() {
             if (!chunk) throw Error(`Cannot find ${id}`);
 
             if (placeholder === mainNamePlaceholder) {
-              return JSON.stringify(
-                chunk.fileName.slice(filenamePrefix.length),
-              );
+              return JSON.stringify(fileNameToURL(chunk.fileName));
             }
 
             return JSON.stringify(
-              getDependencies(chunks, chunk).map((item) =>
-                item.slice(filenamePrefix.length),
-              ),
+              getDependencies(chunks, chunk).map(fileNameToURL),
```

Comment:
```suggestion
              getDependencies(chunks, chunk).map((filename) => fileNameToURL(filename)),
```

https://jakearchibald.com/2021/function-callback-risks/ 😀

Reply:
This function is specifically designed to be a callback though :P

Reply:
It doesn't look like it. It's called in other places not-as-a-callback.

Reply:
Hm. I mean, I specifically wrote it in this form so I could pass it to `.map` (rather than using e.g. object as a param). How do you distinguish between those formats / what should I do to make it more explicit it's intended to be used that way?

Reply:
`fileNameToURLMapper`? Or just use an inline function like above to remove the risk.